tests/smoke.rs

```diff
@@ -77,59 +77,72 @@ fn u128() {
 
 #[test]
 fn f32() {
-    for _ in 0..1000 {
-        let result = fastrand::f32();
-        assert!((0.0..1.0).contains(&result));
-    }
-}
-
-#[test]
-fn f64() {
-    for _ in 0..1000 {
-        let result = fastrand::f64();
-        assert!((0.0..1.0).contains(&result));
-    }
-}
-
-#[test]
-fn digit() {
```

Comment:
Sorry, I accidentally removed several unrelated tests when resolving conflicts. Filed https://github.com/smol-rs/fastrand/pull/110 to re-add them.